app/routes/funding/models/what-funding.js

```diff
@@ -1,3 +1,5 @@
+const { mapStandards } = require('../../../what-funding')
```

Comment:
Attempted to separate responsibility between files.

My understanding is the model should only have information received from the route and produce an object used by the view, hence I have abstracted the functionality of deciding which standards are displayed into a same named JS module.

Is this a valid approach?

Reply:
Yes I agree.  Though I'd maybe tweak the split as there is some properties in the new module which are part of the nunjucks model such as `checked` and `hint`.

```
const standardDetails = {
  'sfi-moorland': {
    text: 'Funding for moorland or rough grazing',
    value: 'sfi-moorland',
    hint: { text: '£6.45 per hectare per year, plus a single payment of £148 per year.' },
    checked: false
  },
  'sfi-arable-soil': {
    text: 'Funding for arable and horticultural soil',
    value: 'sfi-arable-soil',
    hint: { text: '£22 to £40 per hectare per year based on what work you do.' },
    checked: false
  },
  'sfi-improved-grassland': {
    text: 'Funding for improved grassland soil',
    value: 'sfi-improved-grassland',
    hint: { text: '£28 to £58 per hectare per year based on what work you do.' },
    checked: false
  }
}
```

Reply:
Will address this when I change the agreement-calc to inc. moorland and grazing should look similar to:

`
.map(fundingObj => {
      return {
        text: fundingObj.name,
        value: fundingObj.code,
        hint: { text: fundingObj.hint },
        checked: isChecked(selected, fundingObj.code)
      }
`

as I alter the return from the agreement-calc to include these keys